curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-the-date-object-by-building-a-date-formatter/646e482c620eae02878e9d4c.md

```diff
@@ -0,0 +1,215 @@
+---
+id: 646e482c620eae02878e9d4c
+title: Step 21
+challengeType: 0
+dashedName: step-21
+---
+
+# --description--
+
+For the second case, use the value of `mm-dd-yyyy-h-mm`. Inside that case, assign empty template literals to the constant named `currentDateParagraph` using the `textContent` property.
```

Comment:
It looks like we are missing a final step.

For step 22, you need to include the content that goes inside the template strings which would be this
```
${month}-${day}-${year} ${hours} Hours ${minutes} Minutes
```

Reply:
added